### DISCUSSION TITLE
Python attribute Error

### DISCUSSION THREAD

**Discussion by Manishroy1205** [community/community#21458] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/3/e3b07d7839672cf10da6b542bb60d6e78cddf165.png" data-download-href="/uploads/short-url/wuevShqbuG7u2808Ie3YY0K5fqB.png?dl=1" title="Screenshot (409)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/76405048/181064637-23a5241b-6bba-4d65-b784-abe63eb7d31c.png" alt="Screenshot (409)" data-base62-sha1="wuevShqbuG7u2808Ie3YY0K5fqB" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3b07d7839672cf10da6b542bb60d6e78cddf165_2_690x387.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3b07d7839672cf10da6b542bb60d6e78cddf165_2_1035x580.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/3/e3b07d7839672cf10da6b542bb60d6e78cddf165.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/3/e3b07d7839672cf10da6b542bb60d6e78cddf165_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot (409)</span><span class="informations">1366×768 160 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>PLease suggest what is wrong with this code<br>
…</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The definition of the <code>increase</code> function is indented one level too much, so it’s defined <em>inside</em> the <code>Employee</code> constructor (<code>__init__</code>). That makes the function a local variable inside the constructor, not accessible anywhere else.</p>

**Comment by Manishroy1205** [comment]
<p>Thanx Buddy… :grinning:</p>
